modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/actions.js

```diff
@@ -0,0 +1,126 @@
+                        responseStr += temp;
+                    }
+
+                    responseStr = responseStr.slice(0, -1);
+                    var jsonArrObj = JSON.parse('[' + responseStr + ']');
+                    jsonObj[0].data = jsonArrObj;
+
+                    var barChart = new vizg(jsonObj, config);
+                    barChart.draw("#chartA", [{type: "click", callback: callbackmethod}]);
+
+                }
+                else {
+                    jsonObj[0].data = [];
+                    var barChart = new vizg(jsonObj, config);
+                    barChart.draw("#chartA", [{type: "click", callback: callbackmethod}]);
+                }
+            },
+            error: function (xhr, status, error) {
+                var errorJson = eval("(" + xhr.responseText + ")");
+                alert(errorJson.message);
+            }
+        });
+    } else {
+        console.log('Empty User ID list.');
+        alert("User ID list is empty.");
+    }
+}
+
+function selectPickerValChange(selectPickerElement) {
+    var idx = selectPickerElement.options.selectedIndex;
+    if (selectPickerElement.options[idx].value == 'other') {
+        var other = prompt("Please indicate 'other' value:");
+        if (other != '' && isInteger(other)) {
+            var opt = document.createElement('option');
+            opt.value = other;
+            opt.innerHTML = other;
+            selectPickerElement.appendChild(opt);
+            $(selectPickerElement).selectpicker('val', other);
+        } else {
+            selectPickerElement.selectedIndex = 1;
+            $(selectPickerElement).selectpicker("refresh");
+        }
+    }
+}
+
+function isInteger(param) {
+    return (Math.floor(param) == param && $.isNumeric(param));
+}
+
+
+function loadUserList(dropdownId) {
+    var dropdownElementID = '#' + dropdownId;
+
+    $.ajax({
+        type: 'POST',
+        url: '../../bpmn-analytics-explorer/user_id_list',
+        success: function (data) {
+            if (!$.isEmptyObject(dataStr)) {
+                var dataStr = JSON.parse(data);
+                for (var i = 0; i < dataStr.length; i++) {
+                    var opt = dataStr[i].assignUser;
+                    var el = document.createElement("option");
+                    el.textContent = opt;
+                    el.value = opt;
+                    $(dropdownElementID).append(el);
+                }
+                $(dropdownElementID).selectpicker("refresh");
+
+                $.getJSON("/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/js/meta-data-avgExeTimeVsTaskID.json.js", function (result) {
+                    $.each(result, function (i, field) {
+                        jsonObj.push(field);
+                        drawGraph();
+                    });
+                });
+            }
+            else{
+                console.log('Empty User ID list.');
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7